utilities/xmlrpcpp/src/XmlRpcValue.cpp

```diff
@@ -625,11 +625,11 @@ namespace XmlRpc {
             buf[sizeof(buf)-1] = 0;
             os << buf;
           } else { // required_size >= static_cast<int>(sizeof(buf)
-            char required_buf[required_size+1];
-            std::snprintf(required_buf, required_size,
+            std::vector<char> required_buf(required_size+1);
```

Comment:
mind adding `#include <vector>` to this file?

Reply:
No problem. It is added now.